MSAL/src/requests/MSALBaseRequest.m

```diff
@@ -192,9 +188,11 @@ - (void)acquireToken:(nonnull MSALCompletionBlock)completionBlock
          {
              MSALErrorCode code = MSALErrorCodeForOAuthError(oauthError, MSALErrorInteractionRequired);
              
-             [self setEventError:event errorCode:error.code errorDomain:nil];
+             NSError *msalError = MSALCreateAndLogError(_parameters, code, oauthError, tokenResponse.subError, nil, __FUNCTION__, __LINE__, @"%@", tokenResponse.errorDescription);
```

Comment:
CREATE_LOG_ERROR ignores oauthError and subError. I can create an additional macro that would accept those as well, e.g CREATE_LOG_ERROR_WITH_SUBERRORS. Any better name suggestions?